pkg/genericapiserver/genericapiserver.go

```diff
@@ -163,6 +164,13 @@ type GenericAPIServer struct {
 	enableOpenAPISupport   bool
 	openAPIInfo            spec.Info
 	openAPIDefaultResponse spec.Response
+
+	// PostStartHooks are each called after the server has started listening, in a separate go func
```

Comment:
in one separate go func **each** or in one for all? Should be clearer here in the doc